src/pages/salesOfferPackages.tsx

```diff
@@ -136,9 +135,7 @@ const SalesOfferPackages = ({
                                                             className="govuk-label govuk-checkboxes__label"
                                                             htmlFor={`checkbox-${index}-${paymentMethodId}`}
                                                         >
-                                                            {paymentMethods === 'sms'
-                                                                ? upperCase(paymentMethods)
-                                                                : upperFirst(paymentMethods)}
+                                                            {startCase(paymentMethods)}
```

Comment:
We probably still need the sms check since we do still want that to be upperCase

Reply:
as discussed sms not required